src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -909,7 +909,11 @@ export class NotebookService extends Disposable implements INotebookService {
 			standardKernels: [{
 				name: notebookConstants.SQL,
 				displayName: notebookConstants.SQL,
-				connectionProviderIds: [notebookConstants.SQL_CONNECTION_PROVIDER],
+				connectionProviderIds: [
+					notebookConstants.SQL_CONNECTION_PROVIDER,
```

Comment:
I'm a bit confused here. Shouldn't we be checking one of the connection provider registries instead of hardcoding this list?

Reply:
That's probably something we can explore in a separate PR, since it would overcomplicate the fix here.

Reply:
Ok. Can we add a TODO then?

Reply:
https://github.com/microsoft/azuredatastudio/issues/22755

Reply:
Thanks Cory!